.gitlab/deps_fetch.yml

```diff
@@ -7,20 +7,11 @@
   stage: deps_fetch
   needs: []
   script:
-    # HACK: empty the build directory cache (that can come from previous runs)
-    #       to not have remainders of previous runs, which can make our deps bootstrapping logic fail.
-    # TODO: remove this once we switch to k8s runners, they won't have this problem
-    - find "$CI_BUILDS_DIR" ! -path '*DataDog/datadog-agent*' -depth  # -delete implies -depth
-    - find "$CI_BUILDS_DIR" ! -path '*DataDog/datadog-agent*' -delete || true  # Allow failure, we can't remove parent folders of datadog-agent
```

Comment:
I think we are no longer affected by this, since we don't use anything from outside our repo directory now.